### PR TITLE
fixed season creation

### PR DESCRIPTION
**Pull request by bravint**

### CODE REVIEW

File: src/client/components/CreateSeason.jsx [hunk 1]
```diff
@@ -10,38 +10,36 @@ import { TeamForm } from './seasonform/TeamForm';
...
 export const CreateSeason = () => {
     const { state } = useContext(StoreContext);
-    const [token, setToken] = useState()
-    const navigate = useNavigate();
 
-    console.log(state)
+    const navigate = useNavigate();
 
-    const competitionId = state.competitions[0].id;
+    const competitionId = state.selectedCompetition[0].id;
```
Comment (nathanSamosa) [inline src/client/components/CreateSeason.jsx:20]: is `selectedCompetition` still an array containing an object?

**Review by polarbear23 — APPROVED** [review]

**Review by nathanSamosa — APPROVED** [review]

**Review by molloyconor511 — APPROVED** [review]
You need to clean up your act Bravin

File: src/client/components/CreateSeason.jsx [hunk 2]
```diff
@@ -54,50 +52,51 @@ export const CreateSeason = () => {
...
-        navigate(`/${state.user}/${state.competitions[0].id}`)
-    }
+        navigate(`/${state.user}/${competitionId}`, {replace: true});
+    };
 
     const scrollerStyling = {
-        transform: `translate(${800*-(formStep-1)}px, 0)`
-    }
+        transform: `translate(${800 * -(formStep - 1)}px, 0)`,
+    };
```
[the remark this answers is not in the document]
Comment (bravint) [reply src/client/components/CreateSeason.jsx:79]: thank prettier

File: src/client/components/CreateSeason.jsx (hunk 1 above)
Comment (vherus) [reply src/client/components/CreateSeason.jsx:20]: Good shout, Dr. B!

File: src/client/components/CreateSeason.jsx [hunk 3]
```diff
@@ -10,38 +10,36 @@ import { TeamForm } from './seasonform/TeamForm';
...
-    const [formStep, setFormStep] = useState(1)
+        positionMappings: [],
+    });
+    const [formStep, setFormStep] = useState(1);
 
     useEffect(() => {
-        const completedForm = formStep > formTotalSteps
-        if(completedForm) postForm()
-    }, [formStep])
+        const completedForm = formStep > formTotalSteps;
```
Comment (vherus) [inline src/client/components/CreateSeason.jsx:35]: Be careful with variable naming. Bools should be named something that indicates it's a bool, e.g. `isFormCompleted` or `hasCompletedForm`

**Review by vherus — APPROVED** [review]

**Pull request merged**